test/integration/src/site/_collections/index.js

```diff
@@ -10,7 +10,7 @@ describe('_collections', function () {
     });
 
     after(function () {
-      fs.rmdirSync(path.join('.', '.tmp'), {recursive: true});
+      fs.rmdirSync(path.join(__dirname, '.tmp'), {recursive: true});
```

Comment:
This will be the path of this file—are we creating `.tmp` under the test/integration/src/site/_collections path?

Reply:
yes